taggit/managers.py

```diff
@@ -321,10 +328,19 @@ def deconstruct(self):
             kwargs['through'] = self.rel.through
         elif not self.rel.through._meta.auto_created:
             kwargs['through'] = "%s.%s" % (self.rel.through._meta.app_label, self.rel.through._meta.object_name)
-        if isinstance(self.rel.to, six.string_types):
-            kwargs['to'] = self.rel.to
+
+        # rel.to renamed to rel.model in Django 1.9
+        if VERSION >= (1, 9):
+            if isinstance(self.rel.model, six.string_types):
```

Comment:
I get a warning: "Usage of field.rel has been deprecated. Use field.remote_field instead." I think it should be safe to do `self.remote_field.model` here because it's in the 1.9+ branch.